python/foglamp/common/storage_client/payload_builder.py

```diff
@@ -604,8 +607,47 @@ def TIMEBUCKET(cls, timestamp, size="1", fmt=None, alias=None):
 
         return cls
 
+    @classmethod
+    def filter_identifier(cls, identifier):
+        # Characters `(backtick),
+        #            '(single quote)
+        # are not allowed
+        if not isinstance(identifier, str): return identifier
+        if '`' in identifier or "'" in identifier:
+            if cls.action_on_identifier_error == 'halt':
+                raise StorageServerError(code=404, reason="Invalid characters in payload", error=json.dumps(cls.query_payload))
```

Comment:
its not a `StorageServerError`, it should be PayloadBuilderError or May be called `StorageClientError`